doc/user/content/cloud/troubleshooting.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Troubleshooting"
+description: "Troubleshoot issues with Materialize Cloud."
+menu:
+  main:
+    parent: "cloud"
+    weight: 9
+---
+
+{{< cloud-notice >}}
+
+<!-- Do we have a standard set of troubleshooting questions or things we expect people to run into trouble with? If not, I will omit this page for the time being. -->
```

Comment:
OOMs, probably, but we don't have a great story around that right now.

Reply:
After discussion with @awang, I'm pulling this page for now. We can add it back once we have actual troubleshooting advice.